front/temporal/scrub_workspace/activities.ts

```diff
@@ -132,3 +141,113 @@ async function deleteDatasources(auth: Authenticator) {
     }
   }
 }
+
+async function cleanupCustomerio(auth: Authenticator) {
+  const w = auth.workspace();
+
+  if (!w) {
+    throw new Error("No workspace found");
+  }
+
+  // Fetch all the memberships for the workspace.
+  const workspaceMemberships = await MembershipResource.getLatestMemberships({
+    workspace: w,
+  });
+
+  // Fetch all the users in the workspace.
+  const userIds = workspaceMemberships.map((m) => m.userId);
+  const users = (
```

Comment:
You could bundle this change in the MembershipResource conditioned by a `{includeUser: boolean}`.

Reply:
eeeeeeh yeah i could

Reply:
but if this is the only place we'll ever use it, do we want it ?